### DISCUSSION TITLE
How to loop items to send more than 1 notification

### DISCUSSION THREAD

**Discussion by PedroBuffon** [edbrdi/n8n-nodes-discord#14] [Q&A]
Hello guys, i'm having a problem with the discord node that when the node get more than one group of items, it sends only one time and the other items get stuck and are not sent, anyone has any idea how can i loop it to send all the items?
![image](https://user-images.githubusercontent.com/105023308/209863543-a782608b-5f2b-42f4-8ffb-3c6b31efa409.png)
as you guys can see it, in the left side i have multiples items but in the output i only have one message sent.

**Comment by edbrdi** [comment]
Hi, unfortunately at the moment the node doesn't loop over items, but you can use https://docs.n8n.io/integrations/builtin/core-nodes/n8n-nodes-base.splitinbatches/

**Comment by PedroBuffon** [comment]
Ok i think i got it workin with the Split In Batches node, i tried some time before and it didn't work but now with the right setup i got it working, what i did was, first you setup a code node with "return $input.all();" code, that will return all items, after that you setup a batch of 1 to pass only one item at a time into the discord node, after that you loop the discord node back to the batch node, and it will loop all the items you have.
![image](https://user-images.githubusercontent.com/105023308/209881534-d4a938bb-597c-4ad0-bf77-489241337dfe.png)
My final setup is like that.

**Comment by edbrdi** [comment]
Hello, with the next version you won't need to use split in batches: https://github.com/edbrdi/n8n-nodes-discord/commit/0f451bc2fff3389c49887d4a6b5c194a301eab37
